scripts/deploy_build

```diff
@@ -85,7 +86,7 @@ echo \"Installation complete.  Manager can be reached at https://\$(ip addr ls d
 pdsh -R ssh -l root -S -w "$STORAGE_NODES" "set -ex
 # if this node uses the Intel proxies, make sure the agent is doing so
 if [ -f /etc/profile.d/intel_proxy.sh ]; then
-    echo \". /etc/profile.d/intel_proxy.sh\" > /etc/sysconfig/chroma-agent
+    echo -e \"[Service]\\nEnvironmentFile=-/etc/profile.d/intel_proxy.sh\" > /etc/systemd/system/chroma-agent.service.d/override.conf
```

Comment:
Forgot the `mkdir -p /etc/systemd/system/chroma-agent.service.d/` in this one?

Reply:
Yup, I'll fix.